### DISCUSSION TITLE
Save note location information

### DISCUSSION THREAD

**Discussion by dev107** [standardnotes/forum#2607] [Suggestions]
Hi there,

is it possible or planed to add the location to every note? Something like Evernote has: https://help.evernote.com/hc/en-us/articles/209005037-How-to-save-note-location-information

Best wishes

**Comment by bryvin** [comment]
This is not possible currently and not something we have planned. This will remain opened for discussion.

**Comment by blankster** [comment]
That would be very interesting. +1 from me for this proposal.
